pipelines.py

```diff
@@ -306,6 +309,7 @@ def _tokenize(
 
 def pipeline(
     task: str,
+    cpu_only: Optional = False,
```

Comment:
One suggestion, it would be nice if it's named `use_cuda` instead of `cpu_only` to be consistent with libraries, and make it the last parameter for stylistic purpose.  The conditions will also need to be changed little if we make it `use_cuda`. Let me know if your are busy, I'll take it over.